core/src/main/java/de/danoeh/antennapod/core/util/ChapterUtils.java

```diff
@@ -123,6 +138,60 @@ private static CountingInputStream openStream(Playable playable, Context context
         }
     }
 
+    public static List<Chapter> loadChaptersFromUrl(String url) {
```

Comment:
The rest of the class does not do parsing. It just contains the logic for starting the parsers. I think at least the JSON parsing code here would better fit into its own class. Not sure where to put it, maybe the `:parser:feed` module?

Reply:
I moved the code to `model/feed/PodcastIndexChapter.java`, let me know what you think